Source/Revit.IFC.Export/Exporter/Exporter.cs

```diff
@@ -4199,29 +4199,140 @@ private void CreateGlobalCartesianOrigin(ExporterIFC exporterIFC)
          ExporterIFCUtils.SetGlobal2DOriginHandle(origin2d);
       }
 
-      private HashSet<IFCAnyHandle> RemoveContainedHandlesFromSet(ICollection<IFCAnyHandle> initialSet)
+      private static bool ValidateContainedHandle(IFCAnyHandle initialHandle)

```

Comment:
@o-babii : don't we have code that keeps levels now?  This is a workaround but it seems better just not to delete the levels.

Reply:
Yes, we have, it wasn`t backsubmitted because of cpp changes.